src/redux/data.ts

```diff
@@ -573,6 +580,14 @@ export const dataLoadState = () => async (
     try {
       dispatch({ type: DATA_LOAD_TRANSACTIONS_REQUEST });
       const transactions = await getLocalTransactions(accountAddress, network);
+      const pendingTransactions = await getLocalPendingTransactions(
+        accountAddress,
+        network
+      );
+      dispatch({
+        payload: pendingTransactions,
+        type: DATA_ADD_NEW_TRANSACTION_SUCCESS,
+      });
```

Comment:
trying to follow the logic here... Is this action being dispatched to rehydrate redux after  loading txs?

Reply:
as discussed via dm, I think this looks confusing because of the action type's original name. I changed this action's name to `DATA_UPDATE_PENDING_TRANSACTIONS_SUCCESS`.